src/cdk-experimental/menu/menu.ts

```diff
@@ -52,6 +51,8 @@ import {getItemPointerEntries} from './item-pointer-entries';
   exportAs: 'cdkMenu',
   host: {
     '(keydown)': '_handleKeyEvent($event)',
+    '(focus)': '_focusFirstItem()',
+    '[tabindex]': '_getTabIndex()',
```

Comment:
```suggestion
    '[tabindex]': '_isInline() ? 0 : null',
```
It would be acceptable to inline the expression here without creating a new method